actors/builtin/miner/deadline_state.go

```diff
@@ -444,3 +446,161 @@ func (dl *Deadline) popExpiredPartitions(store adt.Store, until abi.ChainEpoch,
 
 	return popped, modified, nil
 }
+
+// RemovePartitions removes the specified partitions, shifting the remaining
+// ones to the left, and returning the live and dead sectors they contained.
+//
+// Returns an error if any of the partitions contained faulty sectors or early
+// terminations.
+func (dl *Deadline) RemovePartitions(store adt.Store, partitions *bitfield.BitField, quant QuantSpec) (
```

Comment:
Suggest `removePartitions` or `toRemove` for the arg, to be more clear when used.